Backend/router/v1/user.js

```diff
@@ -1,27 +1,66 @@
 const express = require('express');
 const router = express.Router();
 const models = require('../../models');
-const crypto = require('crypto')
+const crypto = require('crypto');
+const session = require('express-session');
 
-router.use((req, res, next) => {
+router.use(session({
+  key: 'sid',
+  secret: 'secret',
+  resave: false,
+  saveUninitialized: true,
+  cookie: {
+    maxAge: 24000*60*60
```

Comment:
what the reasoning behind this number?

Reply:
I temporarily chose 24 hours. Could you recommend the time limit?

Reply:
No preference on my side. Just curious why you came up with 24 hours. Most data-sensitive apps (banking apps) have much shorter session, so wanted to see if you gave thought on this.

Reply:
For security and ease of use, I want to set it to 15 minutes.